staging/src/k8s.io/apimachinery/pkg/runtime/converter_test.go

```diff
@@ -36,6 +43,9 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+
+	// Initialize install packages
+	_ "k8s.io/kubernetes/pkg/api/testing"
```

Comment:
@liggitt Goal here is to leverage https://github.com/kubernetes/kubernetes/blob/78a3a4a2b1dae52f8db0b49dd03c6aa8caf806b3/pkg/api/testing/install.go#L19-L44 for use with `legacyscheme.Scheme.AllKnownTypes()` below. Do you happen to know if this the preferred approach, or if there is a better way?